gapic/generator/generator.py

```diff
@@ -54,6 +55,7 @@ def __init__(self, opts: options.Options) -> None:
         self._env.filters['snake_case'] = utils.to_snake_case
         self._env.filters['sort_lines'] = utils.sort_lines
         self._env.filters['wrap'] = utils.wrap
+        self._env.filters['resp_to_response'] = samplegen.resp_to_response
```

Comment:
Bikeshed: This name is a bit weird. It makes me think "response to response". Not having read the code yet, I am not sure what it actually is.

Reply:
Sample configs use the special variable `$resp` to refer to the interesting proto message returned by the API call, i.e. for 'vanilla' methods it's just the return value, for paginated methods it's the repeated element, and so forth. Some part of the sample generation process needs to remove the `$`, and while that's happening it might as well get a less terse name. I really, really didn't want to lump that into the validation logic since it's just a dumb text replacement, so strategic template logic via filter it is.